### DISCUSSION TITLE
7.0.0-alpha.0

### DISCUSSION THREAD

**Discussion by jamesdaniels** [angular/angularfire#2837] [Roadmap]
`ng add @angular/fire@next`

<hr /><em>This discussion was created from the release <a href='https://github.com/angular/angularfire/releases/tag/7.0.0-alpha.0'>7.0.0-alpha.0</a>.</em>

**Comment by akshaybabloo** [comment]
What's changing in this?

**Reply by jamesdaniels** [reply]
Standby, we'll have more comms as we hone in on the design. First and foremost though we'll be supporting Firebase JS SDK v9, currently in beta. We have an entirely new API that is tree-shakable (but not lazy-loaded) and looks more like rxfire, second we'll have the existing AngularFire API (for the most part)  at `@angular/fire/compat/*` which is lazy-loaded but not tree-shakable.

**Reply by anisabboud** [reply]
Is this alpha release usable? Thanks!
